src/components/Cards/ExpertList.js

```diff
@@ -36,43 +42,47 @@ const City = styled.div`
 export default function ExpertList({searchTerm}) {
 	return (
 		<Ul>
-			{data
+			{data.experts
 				.filter(item => {
 					return searchTerm.toLowerCase() === ''
 						? item
 						: item.city.toLowerCase().includes(searchTerm);
 				})
 				.map(item => (
 					<li key={item.id}>
-						<Card>
-							<ImageWrapper>
-								<Image
-									src={
-										item.gender === 'Female'
-											? '/img/avatar-female.jpg'
-											: '/img/avatar-male.jpg'
-									}
-									width={80}
-									height={80}
-									objectFit="cover"
-									alt="portrait"
-								/>
-							</ImageWrapper>
+						<Link href={`/experts/${item.ident}`} passHref>
+							<Anchor>
+								<Card>
+									<ImageWrapper>
+										<Image
+											src={
+												item.gender === 'Female'
+													? '/img/avatar-female.jpg'
+													: '/img/avatar-male.jpg'
+											}
+											width={80}
+											height={80}
+											objectFit="cover"
+											alt="portrait"
+										/>
+									</ImageWrapper>
 
-							<DetailPreview>
-								<div>
-									<text>
-										{item.first_name} {item.last_name}
-									</text>
-									<br />
-									<text>{item.ident}</text>
-								</div>
+									<DetailPreview>
+										<div>
+											<text>
```

Comment:
I never saw this html element before - where did you found this? and why are you not using a `<p> `tag, or a `<span>` - which would be semantically correct?